src/nested_pandas/nestedframe/io.py

```diff
@@ -0,0 +1,69 @@
+# typing.Self and "|" union syntax don't exist in Python 3.9
+from __future__ import annotations
+
+import pandas as pd
+
+from .core import NestedFrame
+
+
+def read_parquet(
```

Comment:
Pandas has `dtype_backend`, but I'm not sure if we'd like to exposure it... We definitely should experiment with `dtype_backend='pyarrow'`, but later. Let's maybe have it here for the completeness and add some TODO for it?

Reply:
I've added the kwarg as is, with the numpy_nullable default as per pandas. I added a todo to investigate the backend more